.eslintrc.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": [
+    "react",
```

Comment:
Is there any reason we are not using `airbnb` styleguide?

Reply:
https://stackoverflow.com/questions/46201647/prettier-airbnbs-eslint-config?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa

Reply:
we will have to discuss this further and come up with a conclusion, personally, I think `airbnb` has created some sort of dogma in the react community whereby guys take it as the golden truth to JS while other libraries like prettier have their own style guide which is also as good in terms of standard.

Reply:
@paradisekelechi I'll like to keep the `.eslintrc` config as simple as possible. How much overriding do we need to do for us to use `airbnb`? If this works across both the presentational and container components and enforces `PropType` validation, Then I'm cool with this.

Reply:
@nwochaadim I agree there are quite a number of overriding. If we want to keep it simple, clean and function, I'd suggest we just use eslint and airbnb.

Reply:
Alright then, I'm sold in on Airbnb